ports/stm32/eth.c

```diff
@@ -259,21 +275,21 @@ static int eth_mac_init(eth_t *self) {
     #if defined(STM32H5)
     __HAL_RCC_ETH_RELEASE_RESET();
 
-    __HAL_RCC_ETH_CLK_SLEEP_ENABLE();
-    __HAL_RCC_ETHTX_CLK_SLEEP_ENABLE();
-    __HAL_RCC_ETHRX_CLK_SLEEP_ENABLE();
+    __HAL_RCC_ETH_CLK_SLEEP_DISABLE();
+    __HAL_RCC_ETHTX_CLK_SLEEP_DISABLE();
+    __HAL_RCC_ETHRX_CLK_SLEEP_DISABLE();
```

Comment:
Is this change really necessary?  I would think that it's OK to just enable the clock sleep feature at the start, and not need to disable/enable it during reset.

Reply:
Having the ENABLE lines the way he was previously is actually what I found prevented .active(True) from working when the ethernet cable was connected (on the H5 at least). The reset just after this would timeout unless the cable was connected.

Reply:
OK, I see.  Maybe it has to do with the WFI hiding in the `mp_hal_delay_ms(2)` (because the loop below that line is a busy loop and does not do WFI).

Well, probably it's OK to just disable clock-sleep altogether, because we do want ETH to run "in the background" when the CPU does WFI.  All other peripherals do that (disable clock-sleep), eg USB.